codelists/api.py

```diff
@@ -170,6 +172,7 @@ def versions(request, codelist):
                 codelist=codelist,
                 codes=set(data["codes"]),
                 tag=data.get("tag"),
+                always_create_new_version=data.get("always_create_new_version", False),
```

Comment:
There should probably be a test for this too  - a version of the `test_versions_post_no_difference` [here](https://github.com/opensafely-core/opencodelists/blob/main/codelists/tests/test_api.py#L153) which passes the `force_create` or `always_create_new_version` flag and does a `with assert_difference`